components/mediation-ui/org.wso2.carbon.endpoint.ui/src/main/java/org/wso2/carbon/endpoint/ui/endpoints/loadbalance/LoadBalanceEndpoint.java

```diff
@@ -96,6 +106,10 @@ public OMElement serialize(OMElement parent) {
         OMElement loadbalance = fac.createOMElement("loadbalance", synNS);
         loadbalance.addAttribute(fac.createOMAttribute("algorithm", nullNS, algorithmClassName));
 
+        if (isBuildMessage()) {
+            loadbalance.addAttribute(fac.createOMAttribute("buildMessage", nullNS, Boolean.toString(true)));
```

Comment:
shall we define the "buildMessage" as a constant ? could you also explain why we hardcode the value here ?